hotfix: remove `cursor: pointer` + invalid dimension on iOS

espeon opened the pull request:
Two fixes:

- Warning on "-$1" not being a valid dimension on iOS

- Fix occasional infinite loading on iOS. So far as I can tell, we can add this back in on the switch to New Architecture eventually.

espeon committed:
fix 'not valid dimension' warning

espeon committed:
fix: can't have cursor: pointer on iOS on Old Architecture

iameli commented on js/app/components/livestream/livestream.tsx:
```diff
@@ -267,7 +267,8 @@ export function LivestreamInner(props: Partial<PlayerProps>) {
                               aria-label={`View @${streamerHandle} on Bluesky`}
-                              style={{ cursor: "pointer" }}
+                              // TODO: re-add on New Architecture
+                              //style={{ cursor: "pointer" }}
```
```suggestion
                              // TODO: re-add on New Architecture
                              style={isWeb ? { cursor: "pointer" } : {}}
```

espeon committed:
Add cursor back on web

Co-authored-by: Eli Mallon <eli@stream.place>

espeon committed:
Merge branch 'next' into natb/hotfix-ios-cursor

espeon committed:
Reverse chat panel sizing regression

espeon committed:
Fix fullscreen regression

- Comparing something to fullscreen that results in a component change
will refresh everything inside the component (!)
